simple-variant-discovery.wdl

```diff
@@ -0,0 +1,199 @@
+version 1.0
+
+workflow SimpleVariantDiscovery {
+    input {
+        File gatk
+        File refFasta
+        File refIndex
+        File refDict
+        String sampleName
+        File inputBam
+        File bamIndex
+        File testando
+    }
+
+    call HaplotypeCaller {
+        input:
+            gatk = gatk,
+            refFasta = refFasta,
+            refIndex = refIndex,
+            refDict = refDict,
+            sampleName = sampleName,
+            inputBam = inputBam,
+            bamIndex = bamIndex
+    }
+
+    call SimpleVariantSelection as selectSNPs { 
+        input:
+            gatk = gatk,
+            refFasta = refFasta,
+            refIndex = refIndex,
+            refDict = refDict,
+            sampleName = sampleName,
+            type = "SNP", 
+                rawVcf = testando
```

Comment:
um recuo a mais.

Reply:
Não entendi, eu to contando a partir do "t" do "type"  e da 4 espaços certinho